packages/perseus/src/widgets/__stories__/expression.stories.tsx

```diff
@@ -48,6 +48,9 @@ const WrappedKeypadContext = ({
                                 isMobile: isMobile,
                                 customKeypad: customKeypad,
                                 onFocusChange: action("onFocusChange"),
+                                interactionCallback: (response) => {
```

Comment:
I found having the console log incredibly helpful for testing the interactionCallback, but I am unsure if we want to present the story in this manner. 

I decided to leave it as is, but I'm open to feedback or alternative opinions on the matter!